internal/config/json_reporter_test.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestJSONReporterSettings(t *testing.T) {
+	type testCaseT struct {
+		conf JSONReporterSettings
+		err  error
+	}
+
+	testCases := []testCaseT{
+		{
+			conf: JSONReporterSettings{Path: "out.json"},
+		},
+		{
+			conf: JSONReporterSettings{},
+			err:  errors.New("empty path"),
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(fmt.Sprintf("%v", tc.conf), func(t *testing.T) {
+			err := tc.conf.validate()
+			if err == nil || tc.err == nil {
```

Comment:
Ideally this should be
```if tc.err == nil {require.NoError(err) //or similar}```

Reply:
This is just copy/paste from another testcase. Happy to keep their style if its not major issue?

Reply:
Sure